tests/src/test/scala/whisk/core/cli/test/WskWebActionsTests.scala

```diff
@@ -250,7 +172,7 @@ trait WskWebActionsTests
                 response.statusCode shouldBe 200
                 response.header("Access-Control-Allow-Origin") shouldBe "*"
                 response.header("Access-Control-Allow-Methods") shouldBe "OPTIONS, GET, DELETE, POST, PUT, HEAD, PATCH"
-                response.header("Access-Control-Allow-Headers") shouldBe "Authorization, Content-Type"
+                response.header("Access-Control-Allow-Headers") shouldBe "Authorization, Content-Type, User-Agent"
```

Comment:
@drcariel, can you split the controller changes out into a separate PR since they are unrelated to the CLI changes?